docs/guides/local_installation.md

```diff
@@ -93,6 +93,6 @@ mkdir -p ~/.terraform.d/plugins/Cox-Automotive/engineering-enablement/alks/1.5.1
 
 ### Supported Versions
 
-| Terraform 0.10.x       | Terraform 0.12.x  | Terraform 0.13.x | Terraform 0.14.x |
-| ----------------       | ----------------  | ---------------- | ---------------- |
-| ALKS TFP 0.9.0 < 1.3.0 | ALKS TFP 1.3.0+   | ALKS TFP 1.3.0+  | ALKS TFP 1.3.0+  |
+| Terraform 0.10.x       | Terraform 0.12.x  | Terraform 0.13.x | Terraform 0.14.x | Terraform 0.11.x |
```

Comment:
Could you move 0.11.x to inbetween 0.10 and 0.12?

Reply:
Ah yeah committed this incorrectly, good catch